### ISSUE TITLE
`explicit-module-boundary-types` complains about setters

### ISSUE THREAD

**Issue by greentore** [denoland/deno_lint#1210]
The linter demands return type on a setter.

### Lint Name

`explicit-module-boundary-types`

### Code Snippet

```ts
export class Test {
  #a = "";
  get a(): string {
    return this.#a;
  }
  set a(v: string) {
    this.#a = v;
  }
}
```

### Expected Result
No warning.

### Actual Result
```
Missing return type on function
Add a return type to the function signature deno-lint (explicit-module-boundary-types)
```
### Version

deno 1.38.0 (release, x86_64-pc-windows-msvc)
v8 12.0.267.1
typescript 5.2.2
